sections/Chapter 7 Sprint 5/Increment.tex

```diff
@@ -0,0 +1,2 @@
+\section{Increment}
+During sprint 5, the project was updated to .NET 6. Following this update, the code quality checker Code QL had to be removed from our GitHub Actions as it does not yet support for .NET 6.
```

Comment:
What is code QL?

Reply:
Code QL is a action mentioned in our CI/CD section